lib/install/config/webpacker.yml

```diff
@@ -43,8 +43,15 @@ development:
     # Inline should be set to true if using HMR
     inline: true
     overlay: true
+    compress: true
     disable_host_check: true
     use_local_ip: false
+    quiet: false
+    headers:
+      'Access-Control-Allow-Origin': '*'
+    watch_options:
+      ignored: /node_modules/
+
```

Comment:
`webpacker.yml` is intended for config that needs to be read by both Ruby and JavaScript (webpack). Seems fine to add additional dev server options here, but we should be careful of bloat.

Reply:
Yeah, just added the important ones that folks would like to change 👍